cedar-policy-validator/src/diagnostics/validation_warnings.rs

```diff
@@ -100,14 +100,16 @@ impl Diagnostic for MixedScriptIdentifier {
 
 /// Warning for identifiers containing confusable characters
 #[derive(Debug, Clone, PartialEq, Error, Eq, Hash)]
-#[error("for policy `{policy_id}`, identifier `{id}` contains characters that fall outside of the General Security Profile for Identifiers")]
+#[error("for policy `{policy_id}`, identifier `{id}` contains the character `{}` which falls outside of the General Security Profile for Identifiers", .confusable_character.escape_debug())]
```

Comment:
should we use `{:?}` instead of `{}`, to print as `Debug` instead of `Display`?  What does this message look like for the newline character `\n`?